src/Cli.Tests/ConfigureOptionsTests.cs

```diff
@@ -150,5 +150,195 @@ public void TestUpdateDepthLimitForGraphQL(int? newDepthLimit)
 
             Assert.AreEqual(newDepthLimit, config.Runtime?.GraphQL?.DepthLimit);
         }
+
+        /// <summary>
+        /// Tests the update of the database type in the runtime config.
```

Comment:
```suggestion
        /// Tests running "dab config -update"
```
something like the above to demonstrate which cli command/usage is tested.